### PR TITLE
PLANNER-2919 Correct Quarkus checks

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/PLANNER-2919

Related:
- https://github.com/kiegroup/optaplanner/pull/2757
- https://github.com/kiegroup/optaplanner-quickstarts/pull/545

### CODE REVIEW

File: .ci/environments/update.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+fi
+
+# If update script is present, apply it
+if [ -f "${env_path}/after.sh" ]; then
+    echo "Run after script"
+    ${env_path}/after.sh $@
+fi
+
+# Download `setup_integration_branch` script and execute
+curl -s https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/dsl/seed/scripts/setup_integration_branch.sh | bash
```
Comment (rsynek) [inline .ci/environments/update.sh:48]: Why do we create any integration branch to update the quarkus version?

Comment (radtriste) [reply .ci/environments/update.sh:48]: optional->script is executed only if `INTEGRATION_BRANCH` env var is present, else it ignores and go on
see also https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/dsl/seed/scripts/setup_integration_branch.sh

Comment (rsynek) [reply .ci/environments/update.sh:48]: If we don't need it, I prefer simplicity.

Comment (radtriste) [reply .ci/environments/update.sh]: removed from both PRs

**Pull request merged**